frontend/src/pages/BYONImages/ImageErrorStatus.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Text, Icon, Tooltip } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { BYONImage } from '~/types';
+
+type ImageErrorStatusProps = {
+  image: BYONImage;
+};
+
+const ImageErrorStatus: React.FC<ImageErrorStatusProps> = ({ image }) => {
+  if (!image.error) {
+    return null;
+  }
+  return (
+    <Tooltip removeFindDomNode role="none" content={<Text>{image.error}</Text>}>
+      <Icon role="button" aria-label="error icon" status="danger" isInline tabIndex={0}>
```

Comment:
I feel this is not the best way to make it tab-enabled... because you're explicitly setting the tab index to 0. We did this elsewhere with a button: https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/components/SupportedAppTitle.tsx#L22-L30